scripts/tts/talk.py

```diff
@@ -71,6 +74,27 @@ def main() -> None:
     nchannels = 1
     sampwidth = 2
     sound_stream, out_f = None, None
+
+    if args.list_voices:
+        config_response = service.stub.GetRivaSynthesisConfig(
+                riva.client.proto.riva_tts_pb2.RivaSynthesisConfigRequest()
+            )
+        tts_models = dict()
+        for model_config in config_response.model_config:
+                language_code = model_config.parameters['language_code']
+                voice_name = model_config.parameters['voice_name']
+                subvoices = [voice.split(':')[0] for voice in model_config.parameters['subvoices'].split(',')]
+                full_voice_names = [voice_name + "." + subvoice for subvoice in subvoices]
+
+                if language_code in tts_models:
+                    tts_models[language_code]['voices'].extend(full_voice_names)
+                else:
+                    tts_models[language_code] = {"voices": full_voice_names}
+
+        tts_models = dict(sorted(tts_models.items()))
+        print(json.dumps(tts_models, indent=4))
+        return
```

Comment:
can we move this at start of main? if both list-voices and list-devices are provided, the list-voices would not honored because of return from line 71